scripts/uberenv/uberenv.py

```diff
@@ -100,6 +102,11 @@ def parse_args():
                       dest="spec",
                       default=None,
                       help="spack compiler spec")
+    # for vcpkg, what architecture to target
+    parser.add_option("--triplet",
+                      dest="triplet",
+                      default=None,
```

Comment:
My intent was to let the option handle command-line user input.  If there was no user input, the triplet option should be `None`.  Further down, we check the environment variable that vcpkg checks, since that is still user input (just not command line input).

Reply:
Thanks @agcapps.  I saw that a default option was added below.

Since this will be new to most of our team/uberenv users, would it make sense to add a link to the possible triplet options? That way they are more easily discoverable from the command line, rather than crawling through the code.